Files/Interacts/Interaction.cs

```diff
@@ -145,6 +145,23 @@ public async void OpenDirectoryInNewTab_Click()
             }
         }
 
+        public async void ItemPointerPressed(object sender, PointerRoutedEventArgs e)
```

Comment:
```suggestion
        public void ItemPointerPressed(object sender, PointerRoutedEventArgs e)
```

Reply:
There is an `Await` so that's required.

Reply:
It is not, if you accept first suggestion.